src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/example-global-adf-cross-organization.yml

```diff
@@ -0,0 +1,341 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "This template is used for boostrapping accounts outside the current Organization"
+Parameters: #make sure before deploying this template that you change the parameters
```

Comment:
```suggestion
Parameters:  # Important note: make sure that you change the parameters before you deploy this template.
```

Reply:
Since this is important, we should add conditions to the parameters below to prevent them from deploying it without changing it.